scripts/consent_form_json.py

```diff
@@ -73,24 +48,23 @@ def get_credentials():
     return credentials
 
 
-def main():
-    credentials = get_credentials()
-    http = credentials.authorize(httplib2.Http())
-    service = discovery.build('drive', 'v3', http=http)
-
-    # get and download files
-    data = service.files().list(q="'" + FOLDER_ID + "'" + " in parents").execute()
+def run(folderid,service):
+    #get and download files
+    data = service.files().list(q="'" + folderid + "'" + " in parents").execute()
     download_files(data['files'], service)
 
     # convert csv files to a single json file
     content = {}
     for filename in os.listdir(files_path):
-        site_id = os.path.splitext(filename)[0]
-        filepath = 'consent_forms/' + filename
-        if not os.path.isfile(filepath):
+        if filename.endswith('.csv'):
+            site_id = os.path.splitext(filename)[0]
```

Comment:
Not necessary to change here, but small convenience note: rather than checking `file.endswith` and then `os.path.splitext(filename)[0]`, you could just write:

```
filename, ext = os.path.splitext(...)
if ext == 'csv':
    ...
```

Both work just fine in this case; python just has a lot of convenience methods  and syntactic sugar to make life easier. :)